doc/nrf/known_issues.rst

```diff
@@ -1203,19 +1290,19 @@ KRKNWK-8842: MPSL does not support nRF21540 revision 1 or older
 802.15.4 Radio driver
 =====================
 
-.. rst-class:: v1-7-1 v1-7-0
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0
```

Comment:
This has been fixed in v1.8.0.

Reply:
OK, I assume the other two were not. Thanks.

Reply:
@czeslawmakarski , what about the nRF5340 one?
`KRKNWK-6756: 802.15.4 Service Layer (SL) library support for the nRF53`

Reply:
@greg-fer Thank you for noticing. This KRKNWK-6756 still stands for NCS 1.8.0.